<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bridges</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="AdjacencyMatrix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterNodesNumb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawGraph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить граф</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NodesNumb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="graphview" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>431</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TextGraphImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изображение заданного графа, ребра-мосты выделены красным цветом</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextEnterNodesNumb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите число вершин графа</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextEnterDistances">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите расстояния между</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextEnterDistances2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вершинами графа</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>graphview</class>
   <extends>QWidget</extends>
   <header>graphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
